README.md

```diff
@@ -22,9 +22,16 @@ var tessel = require('tessel');
 var gpsLib = require('gps-a2235h');
 gpsLib.debug = 0; // switch this to 1 for debug logs, 2 for printing out raw nmea messages
 
-// GPS uses software UART, which is only available on Port C
+// GPS uses software UART, which is only available on Port C on Tessel 1
 // we use Port C because it is port most isolated from RF noise
-var gps = gpsLib.use(tessel.port['C']); 
+// Port C doesn't exist on Tessel 2 so use Port A
+var portToUse = 'C';
+
+if (!tessel.port[portToUse]) {
+  portToUse = 'A';
```

Comment:
Yes, but for the example I chose A to be consistent with all the other example. I'll make the comment more clear.